contracts/common/GetActionSummary.sol

```diff
@@ -140,12 +142,15 @@ contract GetActionSummary is ExtractCallData, GetActionDomainSkillId {
       } else if (_dominantSigs.includes(actionSummary.sig)) {
         // If any of the actions are classed as 'dominant',
         //   the entire multicall is such
-        return actionSummary;
+        totalSummary.sig = actionSummary.sig;
```

Comment:
Why add the tracking boolean vs returning directly?

Reply:
To ensure that invalid multicalls are still caught. I thought the 'disguising' test I added demonstrated that, but I ended up testing slightly different logic there; I've added a second disguising test that, if you return fails.

Reply:
I actually forgot to push this test yesterday, so you can take a look at it now.